ihmc-high-level-behaviors/src/main/java/us/ihmc/perception/RapidHeightMapManager.java

```diff
@@ -29,13 +31,14 @@
  */
 public class RapidHeightMapManager
 {
-   private final RapidHeightMapExtractor rapidHeightMapExtractor;
+   private final RapidHeightMapExtractorCuda rapidHeightMapExtractor;
    private final ImageMessage croppedHeightMapImageMessage = new ImageMessage();
    private final FramePose3D cameraPoseForHeightMap = new FramePose3D();
    private final RigidBodyTransform sensorToWorldForHeightMap = new RigidBodyTransform();
    private final RigidBodyTransform sensorToGroundForHeightMap = new RigidBodyTransform();
    private final RigidBodyTransform groundToWorldForHeightMap = new RigidBodyTransform();
-   private final BytedecoImage heightMapBytedecoImage;
+   private final GpuMat heightMapImage;
+   private final Mat heightmapMat;
```

Comment:
These seem to represent the depth image from a sensor, not really the height map. I would name them something like `deviceDepthImage` and `hostDepthImage`